docs/views/elements/PanelDoc.vue

```diff
@@ -219,6 +219,24 @@
         </Tab>
       </Tabs>
     </div>
+
+    <div class="container">
+      <h2>Admin panel with actions in the header</h2>
+      <p>
+        Instead of a heading element (h2 etc.) as the first child of admin-panel, you can add 
+        a div with a class of 'heading'. The header element inside this div will still be styled as usual. 
+        But you can also add additional content, such as action buttons in the heading div.
+      </p>
+      
+      <div class="admin-panel">
+        <div class="heading bg-primary gradient-info" style="display: grid; grid-template-columns: 1fr auto auto;align-items: center;">
```

Comment:
Can these styles be added to the CSS files. 

`.admin-panel__heading:has(.btn) {
  display: grid; 
  grid-template-columns: 1fr auto auto;
  align-items: center;
}`

Also this should be under a media query as the mobile view is very squashed.